homeassistant/components/incomfort/quality_scale.yaml

```diff
@@ -0,0 +1,69 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: done
+    comment: |
+      Entities are set up dand updated through the datacoordimator.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions: done
+  reauthentication-flow: done
+  parallel-updates: done
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters: done
+
+  # Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default: done
+  discovery: done
+  stale-devices:
+    status: exempt
+    comment: >
+      There is a maximum of 3 heaters that can be discovered by the gateway.
+      The user must remove manually any heeater devices that have been replaced.
```

Comment:
The thing with this rule is that integrations should handle new devices and old devices properly

Reply:
New devices are handled. A climate/room will be replaced automatically. A heater as a serial no, so that could be managed.

Reply:
Solved with https://github.com/home-assistant/core/pull/136566